pkg/models/query.go

```diff
@@ -134,18 +134,19 @@ func (tabularConditions *TwinMakerTabularConditions) ToTwinMakerTabularCondition
 
 // TwinMakerQuery model
 type TwinMakerQuery struct {
-	GrafanaLiveEnabled bool                          `json:"grafanaLiveEnabled,omitempty"`
-	IsStreaming        bool                          `json:"isStreaming,omitempty"`
-	WorkspaceId        string                        `json:"workspaceId,omitempty"`
-	EntityId           string                        `json:"entityId,omitempty"`
-	Properties         []*string                     `json:"properties,omitempty"`
-	NextToken          string                        `json:"nextToken,omitempty"`
-	ComponentName      string                        `json:"componentName,omitempty"`
-	ComponentTypeId    string                        `json:"componentTypeId,omitempty"`
-	PropertyFilter     []TwinMakerPropertyFilter     `json:"filter,omitempty"`
-	ListEntitiesFilter []TwinMakerListEntitiesFilter `json:"listEntitiesFilter,omitempty"`
-	Order              TwinMakerResultOrder          `json:"order,omitempty"`
-	MaxResults         int                           `json:"maxResults,omitempty"`
+	GrafanaLiveEnabled   bool                          `json:"grafanaLiveEnabled,omitempty"`
+	IsStreaming          bool                          `json:"isStreaming,omitempty"`
+	WorkspaceId          string                        `json:"workspaceId,omitempty"`
+	EntityId             string                        `json:"entityId,omitempty"`
+	Properties           []*string                     `json:"properties,omitempty"`
+	PropertyDisplayNames map[string]string             `json:"propertyDisplayNames,omitempty"`
```

Comment:
```suggestion
        // Optional metadata saved with the query.  When this matches properties used in the results, it will
        // replace the display name
	PropertyDisplayNames map[string]string             `json:"propertyDisplayNames,omitempty"`
```

or something like that 🤷